### PR TITLE
fix labels on new user prompt

### PR DESCRIPTION
**Pull request by tcitworld**
This shows proper labels on the new user creation panel.

### CODE REVIEW

File: src/Wallabag/CoreBundle/Resources/translations/validators.fr.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Config screen
+The password fields must match: 'Les deux mots de passe doivent être les mêmes'
```
Comment (tcitworld) [inline src/Wallabag/CoreBundle/Resources/translations/validators.fr.yml]: Does french still has those accents ? Just saying.

Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/translations/validators.fr.yml]: :trollface: :100:

**Comment by j0k3r** [comment]
What was the problem?

**Comment by tcitworld** [comment]
The problem was that the two passwords label were simply *First* and *Second*. Also, the error message wasn't translated.

I realize the length constraint on the password doesn't seem to work. Looking into it.

File: src/Wallabag/CoreBundle/Controller/ConfigController.php [hunk 2]
```diff
@@ -124,7 +124,6 @@ public function indexAction(Request $request)
         // enable created user by default
         $newUser->setEnabled(true);
         $newUserForm = $this->createForm(NewUserType::class, $newUser, array(
-            'validation_groups' => array('Profile'),
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Controller/ConfigController.php]: I think it's still required for FOSUser

Comment (tcitworld) [inline src/Wallabag/CoreBundle/Controller/ConfigController.php]: Seems so. But the length assertion still fails.

**Comment by j0k3r** [comment]
Could you squash your commits?

**Pull request merged**